revscoring/utilities/union_merge_observations.py

```diff
@@ -40,36 +41,21 @@ def main(argv = None):
     else:
         out_file = open(args['--output'], "w")
 
-    transformation = DataUnion(id_column = args['--id-column'])
-    transformation.union(args['<input>'], out_file)
+    observations = []
+    for path in args['<input>']:
+        with open(path, "r") as in_file:
+            observations.extend(read_observations(in_file))
 
+    union_merge(observations, out_file, id_column = args['--id-column'])
 
-class DataUnion(object):
 
-    # FIXME: This doubled default is annoying.
-    def __init__(self, id_column = "rev_id"):
-        self.id_column = id_column
+def union_merge(observations, out_file, id_column):
```

Comment:
Including the out_file here seems problematic.  It seems that we should have a function for doing the in-memory operations and then we can separately decide to write output to a file.